app/src/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -339,11 +343,21 @@ protected void addQuestionText(final FormEntryPrompt p) {
 
         // Add the text view. Textview always exists, regardless of whether there's text.
         mQuestionText = new TextView(getContext());
-        mQuestionText.setText(p.getLongText());
-        mQuestionText.setTextSize(TypedValue.COMPLEX_UNIT_DIP, mQuestionFontsize);
-        mQuestionText.setTypeface(null, Typeface.BOLD);
-        mQuestionText.setPadding(0, 0, 0, 7);
         mQuestionText.setId(38475483); // assign random id
+        
+        // if we have markdown, use that. 
+        if(markdownText != null){
+            mQuestionText.setText(MarkupUtil.styleSpannable(getContext(), markdownText));
+            mQuestionText.setMovementMethod(LinkMovementMethod.getInstance());
+        } else {
+            mQuestionText.setText(p.getLongText());
+            mQuestionText.setTextSize(TypedValue.COMPLEX_UNIT_DIP, mQuestionFontsize);
```

Comment:
Does the default markdown text size match this? Is there a way to make that the case if not?